app/views/idv/shared/_ssn.html.erb

```diff
@@ -38,8 +42,13 @@
 
   <p><%= flow_session[:error_message] %></p>
 
-  <button type="submit" class="display-block margin-y-5 usa-button usa-button--big usa-button--wide">
+  <button type="submit" class="usa-button usa-button--big usa-button--wide">
```

Comment:
Based on the Figma designs for this flow, I think we'll want to keep the margin styles that had existed in one of the partials, showing as removed here.

```suggestion
  <button type="submit" class="display-block margin-y-5 usa-button usa-button--big usa-button--wide">
```

Reply:
You're right, nice catch. And I think we want this for both the case when the user is entering their SSN for the first time and also when they're changing their SSN (previously it was only on the update page)

Reply:
> You're right, nice catch. And I think we want this for both the case when the user is entering their SSN for the first time and also when they're changing their SSN (previously it was only on the initial SSN page)

Yep, I think so. Another win for consolidation 😄

Reply:
Added those styles back in da36d11064149805774a30684df69fbc3e1a06aa